app/src/main/assets/country_metadata/MH.yml

```diff
@@ -1,4 +1,5 @@
 # Do not edit. Source files are in /res/country_metadata
+measurementSystem: [imperial, metric]
```

Comment:
The same reason why I added Palau (PW): because the [Marshall Islands](https://en.wikipedia.org/wiki/Marshall_Islands) were a territory of the US but gained independence about 30 years ago but they are still in a conversion from imperial to metric units and traffic signs...